README.md

```diff
@@ -61,13 +61,47 @@ Build and test Python code:
 
 Best and test Go code:
 
-    go test -v ./go/... 
+    go test -v ./go/...
 
 
 ## Dependencies ##
 
  - A working C++11 compiler.
 
+ - autoconf/automake etc.
+ - cmake
+ - git
+ - GNU make
+ - libtool
+ - Tcl
+ - pkgconf
+ - python27
+ - [depot_tools](https://www.chromium.org/developers/how-tos/install-depot-tools)
+
+## Building with gclient ##
+
+This is the recommended method for all platforms as it gives you a reproducible
+build.
+
+Known to work on FreeBSD 10, OS X (10.10) [tested with XCode + brew installation
+of above deps], and Ubuntu 14.04.
+
+    $ mkdir ct  # or whatever directory you prefer
+    $ cd ct
+    $ gclient config git@github.com:google/certificate-transparency.git
```

Comment:
Add note (somewhere) that if you want to clone from a branch, for example, then you need to do:

`gclient config --name="certificate-transparency" git@github.com:google/certificate-transparency.git@branch`